bftengine/src/bftengine/MsgsCommunicator.cpp

```diff
@@ -49,11 +49,14 @@ void MsgsCommunicator::stopMsgsProcessing() {
   LOG_INFO(GL, "Messages processing for replica " << replicaId_ << " stopped");
 }
 
-int MsgsCommunicator::sendAsyncMessage(NodeNum destNode, char* message, size_t messageLength) {
-  return communication_->send(destNode, std::vector<uint8_t>(message, message + messageLength));
+int MsgsCommunicator::sendAsyncMessage(NodeNum destNode, char* message, size_t messageLength, NodeNum destEndpointNum) {
+  return communication_->send(destNode, std::vector<uint8_t>(message, message + messageLength), destEndpointNum);
 }
 
-void MsgsCommunicator::send(std::set<NodeNum> dests, char* message, size_t messageLength) {
+void MsgsCommunicator::send(std::set<NodeNum> dests,
+                            char* message,
+                            size_t messageLength,
+                            std::set<NodeNum> destEndpoints) {
```

Comment:
No need to change the API. We can rely on a default ICommunication destEndpointNum value in both functions.

Reply:
done.